index.js

```diff
@@ -0,0 +1,32 @@
+const readMore = () => {
+    const dots = document.getElementById("dots");
+    const moreText = document.getElementById("more");
+    const btnText = document.getElementById("readBtn");
```

Comment:
We should be using querySelector. getElementById is becoming more and more outdated.